exchanges.yml

```
@@ -245,10 +245,10 @@ components:
           description: The step from the above set that the exchange starts on. Passing intialStep is REQUIRED.
         controller:
           type: string
-          description: The controller of the instance. Passing controller is OPTIONAL.
+          description: An OPTIONAL property that specifies the root controller of the instance, which can be used by systems supporting Authorization Capabilities (ZCAPs). This value can be used in conjunction with the <code>authorization</code> property to simultaneously allow other authorization mechanisms.
```

@msporny Thanks for creating a PR for this issue. Since `controller` could potentially be used for any authorization mechanism based on object capabilities, we may want to change the wording slightly:

```suggestion
          description: An OPTIONAL property that specifies the root controller of the instance, which can be used by systems supporting authorization mechanisms like Authorization Capabilities (ZCAPs), which rely on object capabilities. This value can be used in conjunction with the <code>authorization</code> property to simultaneously allow other authorization mechanisms.
```